green-hub/src/components/Home/Home.jsx

```diff
@@ -0,0 +1,170 @@
+import styles from "../Home/Home.module.css";
+import { Link } from "react-router-dom";
+import useAuth from "../../context/AuthContext";
+import copywriting from "../../../assets/images/page-main/copywriting.png";
+import heart from "../../../assets/images/page-main/heart.png";
+import idea from "../../../assets/images/page-main/idea.png";
+import trophy from "../../../assets/images/page-main/trophy.png";
+
+import { getDocs, collection } from "@firebase/firestore";
+import { db } from "../../config/firebase";
+import { useState, useEffect } from "react";
+import Background from "../Background/Background";
+import Footer from "../Footer/Footer"
+import DisplayPoints from "../DisplayPoints/DisplayPoints";
+import { UseUserPoints } from "../Utils/UseUserPoints/UseUserPoints";
+
+const Home = () => {
+  const { logout, currentUser } = useAuth();
+  const [user, setUser] = useState("");
+  const usersCollectionRef = collection(db, "users");
+
+  const getUser = async () => {
+    try {
+      const data = await getDocs(usersCollectionRef);
+      const filteredData = data.docs.map((doc) => ({
+        ...doc.data(),
+      }));
+      const user = filteredData.filter(
+        (user) => user.email === currentUser.email
+      );
```

Comment:
najlepiej to by było uderzać do firebase'a już po konkretnego usera :) może dałoby się pobierać określony dokument, a nie kolekcję?